docarray/documents/video.py

```diff
@@ -97,12 +99,30 @@ class MultiModalDoc(BaseDoc):
     ```
     """
 
-    url: Optional[VideoUrl]
-    audio: Optional[AudioDoc] = AudioDoc()
-    tensor: Optional[VideoTensor]
-    key_frame_indices: Optional[AnyTensor]
-    embedding: Optional[AnyEmbedding]
-    bytes_: Optional[VideoBytes]
+    url: Optional[VideoUrl] = Field(
+        description='URL to the remote video file that needs to be loaded',
+        example='https://github.com/docarray/docarray/blob/main/tests/toydata/mov_bbb.mp4?raw=true',
+    )
+    audio: Optional[AudioDoc] = Field(
+        description='Audio document document associated with the video',
```

Comment:
@JoanFM Please let me know if this is a valid description and also an example for the field. Everything I could think of required a multiple line example